opentreemap/treemap/search.py

```diff
@@ -266,9 +262,12 @@ def fn(predicate_value, field=None):
             if isinstance(value, datetime):
                 date_value = value.date().isoformat()
                 inner_value = {field: date_value}
+            elif isinstance(value, float):
+                inner_value = {field: value}
             else:
-                raise ParseException("Cannot perform min/max comparisons on "
-                                     "non-date hstore fields at this time.")
+                raise ParseException(
+                    "Cannot perform min/max comparisons on "
+                    "non-date/numeric hstore fields at this time.")
```

Comment:
I can see how this is unclear. Integers will never make it this far, because `_parse_value` is called above and will successfully cast integers to floats. So dates become dates and are handled, integers and floats become floats and are handled. In other words, this `ParseException` should only occur if you pass a string that fails to parse into a date or float.